### ISSUE TITLE
PyQt5 support

### ISSUE THREAD

**Issue by PJ-Finlay** [ronaldoussoren/py2app#343]
I'm trying to bundle a basic PyQt application with py2app with no other dependencies (eventually I want to package [Argos Translate](https://github.com/argosopentech/argos-translate)). I'm able to build successfully but when I try to run my application doesn't respond indefinitely. 

- macOS Big Sur 11.2.3
- PyQt5==5.15.1
- py2app==0.23

### [Full code](https://github.com/PJ-Finlay/pyqt-starter/tree/e6c015029bfe5d513fd30c2164be3346f17b4dbf)

Is packaging a PyQt application with py2app possible? Are there other good ways to do it?

Thanks

Edit: Updated "Full code" link to fix bug

**Comment by ronaldoussoren** [comment]
Packaging PyQt5 applications using py2app is supposed to work, including some code to deal with PyQt5 specifically (and PyQt4, PySide and PySide2). 

I'll try to reproduce later.

BTW. Thanks for providing a minimal app, that makes reproducing and fixing a lot easier.

**Comment by ronaldoussoren** [comment]
What doesn't work for you?

I've tested with the repository version of py2app, Python 3.8 and the latest PyQt5. Launching the "my_application" example works for me, both from the Terminal and using the Finder. 

I tried to reproduce with Python 3.9 as well, but failed because I'm running a "universal2" build of Python 3.9 and the PyQt5 wheel is not compatible with that (pip tries to build from source, even in an x86_64 VM).

All of this is using a Python installer from www.python.org.
